src/pybind/mgr/mgr_module.py

```diff
@@ -299,6 +300,12 @@ def _parse_args(self):
                     arg_d[k] = v
                 else:
                     self.args_dict[v] = arg_d
+                if k == 'kw' and v == 'true':
```

Comment:
I find the `kw` keyword a bit too weird, especially since we're then dropping "kwarg" terms bellow. I find going with something a bit more descriptive (especially since we're using this on the CLI descriptor) would be ideal. Something like "requires_args" or "with_map_args". More to type, but I'm sure with a bit of creativity we can find a better keyword ;)

Reply:
maybe `k/v` argument? kv_arg?

Reply:
`kw` stands for keyword, borrowed from pythons [keywords arguments](https://docs.python.org/3/tutorial/controlflow.html#keyword-arguments). `kwarg` might be better and should be familiar to python developers.

Reply:
but if we are exposing `kw` as a thing to the user (like we do on returning EINVAL later in the patch), this is no longer just for developer consumption and, I'd say, a more relatable term might be better.

Reply:
Yeah fair point.